docs/source/configuration/volto-slate/index.md

```diff
@@ -0,0 +1,55 @@
+---
+html_meta:
+  'description': 'What is Volto slate'
+  'property=og:description': 'What is Volto slate'
+  'property=og:title': 'Volto slate'
+  'keywords': 'Volto, Plone, frontend, React, slate, volto slate'
+---
+
+# Volto slate
+
+An interactive default text editor for Volto, developed on top of [slate](https://docs.slatejs.org/) which offers enhanced WYSIWYG functionality and behavior. See a brief elevator pitch for volto-slate: https://youtu.be/SOz-rk5e4_w
+
```

Comment:
All the following should be one sentence per line. I will also make additional suggestions below.

Reply:
Resolved.